src/main/java/codeu/model/store/basic/UserStore.java

```diff
@@ -57,6 +57,24 @@ public static UserStore getTestInstance(PersistentStorageAgent persistentStorage
 
   /** The in-memory list of Users. */
   private List<User> users;
+  
+  /** Access the current amount of users **/
+  public int getNumUsers() {
+	  return users.size();
+  }
+  
+  public String getNewest() {
```

Comment:
This method isn't tested; remove it from this PR, since you're not using it, or test it please